node/modules/proplogcleaner.go

```diff
@@ -0,0 +1,53 @@
+package modules
+
+import (
+	"context"
+	"time"
+
+	"github.com/filecoin-project/boost/db"
+	logging "github.com/ipfs/go-log/v2"
+	"go.uber.org/fx"
+)
+
+var dplcLog = logging.Logger("dplc")
+
+// Boost inserts a row into the DB for each deal proposal accepted or rejected.
+// This method periodically cleans up the rows.
+func HandleProposalLogCleaner(duration time.Duration) func(lc fx.Lifecycle, plDB *db.ProposalLogsDB) {
+	return func(lc fx.Lifecycle, plDB *db.ProposalLogsDB) {
+		var cancel context.CancelFunc
+		var cleanerCtx context.Context
+
+		run := func() {
+			frequency := time.Minute
+			dplcLog.Infof("Starting deal proposal log cleaner to clean up logs older than %s every %s", duration, frequency)
```

Comment:
I think this will be rather spammy. Maybe downgrade to `Debugf` ?
```suggestion
			dplcLog.Debugf("Starting deal proposal log cleaner to clean up logs older than %s every %s", duration, frequency)
```